xml/warewulf.xml

```diff
@@ -0,0 +1,228 @@
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Writes the required PXE files to the TFTP root directory and enables the TFTP service.
+     </para>
+    </listitem>
+   </itemizedlist>
+  </step>
+  <step>
+   <para>
+    Import a compute node container from the &suse; registry:
+   </para>
+<screen>&prompt.user;sudo wwctl container import \
+docker://registry.suse.com/suse/containers/<remark>TBD</remark> \
+<remark>TBD</remark> --setdefault</screen>
+   <para>
+    The <option>--setdefault</option> argument sets this as the default container
+    in the <literal>default</literal> node profile.
+   </para>
+  </step>
+  <step>
+   <para>
+    Configure the networking details for the <literal>default</literal> node profile:
+   </para>
+<screen>&prompt.user;sudo wwctl profile set -y default --netname default \
+--netmask &subnetmask; --gateway &subnetI;.1</screen>
+   <para>
+    To see the details of this profile, run the following command:
+   </para>
+<screen>&prompt.user;sudo wwctl profile list -a</screen>
+  </step>
+  <step>
+   <para>
+    Add compute nodes to &warewulf;. For example, to add ten discoverable nodes
+    with preconfigured IP addresses, run the following command:
+   </para>
+<screen>&prompt.user;sudo wwctl node add n[01-10] \ <co xml:id="co-warewulf-deploy-node-name"/>
+--netdev eth0 -I &subnetI;.11 \ <co xml:id="co-warewulf-deploy-node-network"/>
+--discoverable=true <co xml:id="co-warewulf-deploy-node-discover"/></screen>
+   <calloutlist>
+    <callout arearefs="co-warewulf-deploy-node-name">
+     <para>
+      One or more node names. Node names must be unique. If you have node groups
+      or multiple clusters, add descriptors to the node names, for example
+      <literal>n01.cluster01</literal>.
+     </para>
+    </callout>
+    <callout arearefs="co-warewulf-deploy-node-network">
+     <para>
+      The IP address for the first node. Subsequent nodes will be given incremental
+      IP addresses.
+     </para>
+     <para>
+      <remark>Should this address be inside or outside the DHCP range? In the
+       internal doc it's inside the range but in the upstream doc it's outside the range.</remark>
+     </para>
+    </callout>
+    <callout arearefs="co-warewulf-deploy-node-discover">
+     <para>
+      Allows &warewulf; to assign a MAC address to the nodes when they boot for
+      the first time.
+     </para>
+    </callout>
+   </calloutlist>
+   <para>
+    To view the settings for these nodes, run the following command:
+   </para>
+<screen>&prompt.user;sudo wwctl node list -a n[01-10]</screen>
+  </step>
+  <step>
+   <para>
+    Build the default system and runtime overlays for each node:
+   </para>
+<screen>&prompt.user;sudo wwctl overlay build</screen>
+   <para>
+    <remark>Do we need to do this, or is it automatic and you only need to run
+     the command if you edit any overlays?</remark>
+   </para>
```

Comment:
The overlays depend as well on the configuration values of the nodes as on the overlay template files itself. warewulfd tries to track changes, but fails often to do this.
So to be sure add this call.